drivers/serial/uart_native_tty_bottom.c

```diff
@@ -244,3 +327,23 @@ int native_tty_configure_bottom(int fd, struct native_tty_bottom_cfg *cfg)
 
 	return 0;
 }
+
+int native_tty_config_get_bottom(int fd, struct native_tty_bottom_cfg *cfg)
```

Comment:
This and the ones it calls are now unused. Should they just not be dropped?

Reply:
oh, of course.
surprisingly I don't get a warning about unused function (`west build -b native_sim . --pristine`)